verifier.go

```diff
@@ -22,9 +22,21 @@ type Verifier interface {
 	Verify(content, signature []byte) error
 }
 
+// DigestVerifier is an interface for public keys to verify digested COSE signatures.
+type DigestVerifier interface {
+	Verifier
```

Comment:
Does `DigestVerifier` require `Verify(content, signature []byte) error`?

If no, it should be replaced with
```suggestion
	// Algorithm returns the signing algorithm associated with the public key.
	Algorithm() Algorithm
```

Reply:
Not really, good catch.